database/schema.sql

```diff
@@ -29,10 +29,13 @@ CREATE TABLE public.users (
   id VARCHAR PRIMARY KEY NOT NULL UNIQUE CHECK (id <> ''),
   nonce UUID UNIQUE,
   avatar_url VARCHAR CHECK (avatar_url <> ''),
-  cover_photo_url VARCHAR CHECK (cover_photo_url <> ''),
-  email VARCHAR UNIQUE CHECK (email <> ''),
   username VARCHAR UNIQUE CHECK (username <> ''),
-  preferences VARCHAR UNIQUE CHECK (preferences <> ''),
+  name VARCHAR CHECK (name <> ''),
+  description VARCHAR CHECK (description <> ''),
+  title VARCHAR CHECK (title <> ''),
+  status VARCHAR CHECK (status <> ''),
+  location VARCHAR CHECK (location <> ''),
+  website VARCHAR CHECK (website <> ''),
```

Comment:
We don't need to have the `website` attribute to be in the `users` table since we'll use the table `links` that has a polymorphic relationship between to `users` and to `organizations`, so the website link will be placed in there.

The `linkable_id` will be referencing to the `users`'s primary key and the `linkable_type` will be the name of the table -> `users`

Reply:
@carlomigueldy Changes made for user table